tests/caasp/transactional_update.pm

```diff
@@ -74,17 +74,11 @@ sub run {
     check_package rpmver('in');
 
     record_info 'Update #1', 'Add repository and update - snapshot #2';
-    if (is_caasp 'caasp') {
-        # Only CaaSP needs an additional repo for testing
-        assert_script_run 'zypper ar utt.repo' if is_caasp 'caasp';
-        trup_call 'cleanup up';
-        check_reboot_changes;
-        check_package rpmver('up');
-    }
-    else {
-        # No update repositories ATM
-        record_soft_failure('Test skipped on openSUSE Kubic - bsc#1069784');
-    }
+    # Only CaaSP needs an additional repo for testing
+    assert_script_run 'zypper ar utt.repo' if is_caasp 'caasp';
```

Comment:
May you could try to use `zypper_call` for the next time (available in `lib/utils.pm`).

PS: I didn't click at time for sending my review before the merge happen :)

Reply:
Right, I forgot about it in transactional-update context :) - thanks